pom-gwt.xml

```diff
@@ -122,41 +122,16 @@
     <dependency>
       <groupId>com.google.gwt</groupId>
       <artifactId>gwt-user</artifactId>
-      <version>2.8.0-SNAPSHOT</version>
+      <version>2.8.0-beta1</version>
     </dependency>
 
     <dependency>
       <groupId>com.google.gwt</groupId>
       <artifactId>gwt-dev</artifactId>
-      <version>2.8.0-SNAPSHOT</version>
+      <version>2.8.0-beta1</version>
     </dependency>
   </dependencies>
 
-  <repositories>
```

Comment:
We still need this section, I would think? Guava 20.0-SNAPSHOT is not published on https://repo1.maven.org/maven2/com/google/guava/guava/ it looks like.